Toolset/resources/supporting_files/property_definitions/com.livecode.interface.classic.Stack.txt

```diff
@@ -7,7 +7,7 @@ name
 title	
 mainStack		
 decorations												
-style							Toplevel,Modeless,Modal,Palette
+style							Toplevel	Toplevel,Modeless,Modal,Palette
```

Comment:
If you do `style::revIDESetModeOfStack` here, it will use the `revIDESetModeOfStack` as the property setter, so you can remove your change to `revIDEPropertySet` (in theory)